### PR TITLE
Support for autoscaling by more than 1 instance #22

### PR DESCRIPTION
**Pull request by rpliva**
Added two new labels which allow to set count of instances to scale up or down. Both are optional and backward compatible (default is 1)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ services:
     ports:
       - "5672:5672"
       - "15672:15672"
+  rabbitmq-second:
```
Comment (alenkacz) [inline docker-compose.yml]: why did you added this? @rpliva

**Review by alenkacz — COMMENTED** [review]
Otherwise looking good to me. Thanks! 👍

File: docker-compose.yml (hunk 1 above)
Comment (rpliva) [reply docker-compose.yml]: It was probably the rest of the past. I removed it.

**Comment by alenkacz** [comment]
@rpliva 2.3.0 is being released right now :)

**Pull request merged**